### DISCUSSION TITLE
Rencontre 27/6

### DISCUSSION THREAD

**Discussion by claraquintela** [582-41E-MA/projet-web-2-CollRen#39] [Announcements]
Bonjour!
Notre daily va être ou? :)

**Comment by CollRen** [comment]
On s'envoie un lien dans cette discussion-ci? @claraquintela @NabilDjerroud

**Comment by claraquintela** [comment]
oui. mais hier tu a mencioné zoom. Je n'ai pas de zoom, seulement hangout.

Em qui., 27 de jun. de 2024 às 10:33, René de Montigny <
***@***.***> escreveu:

**Reply by claraquintela** [reply]
lien hangout: 
https://meet.google.com/tnp-krga-ebe

**Reply by CollRen** [reply]
On y va sur zoom pour tester l'enregistrement!
https://us05web.zoom.us/j/82048986152?pwd=wbdgg3EJZhh72zZPUEJObY0LixHpEb.1

**Reply by NabilDjerroud** [reply]
Bonjour @claraquintela @CollRen , désolé les gars je ne pourrais pas assister a cette rencontre je suis dehors avec mes parents
